components/suggest/src/db.rs

```diff
@@ -69,6 +69,12 @@ impl From<ConnectionType> for OpenFlags {
     }
 }
 
+#[derive(Default, Clone)]
+pub struct Sqlite3Extension {
+    pub library: String,
+    pub entry_point: String,
```

Comment:
Maybe we can make `entry_point` optional, as per SQLite's [doc](https://www.sqlite.org/loadext.html), it will figure out the entry point on its own if unspecified. Consumers only need to specify it for other special cases.